### DISCUSSION TITLE
psacc does not read the info.db after restart

### DISCUSSION THREAD

**Discussion by rickcrash** [flobz/psa_car_controller#556] [General]
Hi,

I am using psacc for about a week.

I have recorded about 15 routes and 7 charge-events.

When I restart psacc, the recorded data is not shown.

There is written `No data to show, there is probably no trips recorded yet`

I am running debian Linux an start the psacc as systemd.service:

```
[Unit]
Description=PSA Controller
After=network-online.target

[Service]
ExecStart=/usr/local/bin/psa-car-controller -l 0.0.0.0 -f config.json -c charge_config.json -r -d -R 60
WorkingDirectory=/home/dietpi
Restart=on-failure
RestartSec=1m
Environment=PYTHONUNBUFFERED=true
User=dietpi

[Install]
WantedBy=multi-user.target

```
Any hints?

Thank you in advance
Rick

**Comment by FEmser** [comment]
I got the same issue. Played around a little and found that there seems to be a problem with the "position" table in info.db.
If I remove all rows, the dashboard and charge tab load normally. If i put just one line in there, it will show "No data to show, there is probably no trips recorded yet" on all tabs.

**Reply by sod75** [reply]
Same for me after updating to v3.0.12 (docker) , I tried reimporting the lines in position table from an earlier file but it still didn't work.

 I deleted the db and it recreates an empty new one. I didn't have much in there, so not the end of the world
